src/test/kotlin/no/risc/MigrationFunctionTests.kt

```diff
@@ -0,0 +1,126 @@
+package no.risc
+
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.jsonArray
+import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.json.jsonPrimitive
+import no.risc.risc.ContentStatus
+import no.risc.risc.RiScContentResultDTO
+import no.risc.risc.RiScStatus
+import no.risc.utils.migrate
+import no.risc.utils.migrateFrom33To40
+import no.risc.utils.migrateTo32To33
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Test
+import java.io.File
+
+class MigrationFunctionTests {
+    private val latestSupportedVersion = "4.0"
+
+    @Test
+    fun `test migrateFrom33To32`() {
+        val resourcePath = "3.2.json"
+        val resourceUrl = object {}.javaClass.classLoader.getResource(resourcePath)
+        val file = File(resourceUrl!!.toURI())
+        val fileContent = file.readText()
+        val obj =
+            RiScContentResultDTO(
+                riScId = "1",
+                status = ContentStatus.Success,
+                riScContent = fileContent,
+                riScStatus = RiScStatus.Published,
+                migrationChanges = false,
+            )
+        val migratedObject = migrateTo32To33(obj)
+
+        val json = Json { ignoreUnknownKeys = true }
+        val migratedJsonObject = json.parseToJsonElement(migratedObject.riScContent!!).jsonObject
+
+        val schemaVersion = migratedJsonObject["schemaVersion"]?.jsonPrimitive?.content
+
+        assertEquals("3.3", schemaVersion)
+    }
+
+    @Test
+    fun `test migrateFrom33To40`() {
```

Comment:
Det mangler ein test! Har du gløymt kvifor eg ba deg legge inn tester? 😇 

Lag ein test for 33To40 funksjonen der scenarios mangler i content 👍

Reply:
Oops!